### ISSUE TITLE
Update project status

### ISSUE THREAD

**Issue by Yukaii** [hackmdio/hackmd-desktop#16]

**Comment by RoyiAvital** [comment]
Is this project alive?
Does it support working with local files on the HD?

**Comment by mesqueeb** [comment]
Is this project official? How can I know it's safe?

**Comment by Yukaii** [comment]
@mesqueeb Yes, this is one of our official projects. Due to the limited development resource in our team, we do not actively maintain it recently. 

For the security concerns, you can grab the installer from repo’s releasing page, or even build it from source code.

**Comment by Yukaii** [comment]
@RoyiAvital Sorry for the late reply. Currently only editing notes on official instance(hackmd.io) or self hosted CodiMD server are possible.

**Comment by RoyiAvital** [comment]
@Yukaii , It would be great to have ability to edit local files with your engine.
Could you partner with @PortableApps on this?
It would be great addition to their suite and will bring you exposure to new users.
